rewrite.yml

```diff
@@ -65,6 +65,34 @@ recipeList:
   #  - org.openrewrite.java.cleanup.ReplaceLambdaWithMethodReference
   # Crashes
   #  - org.openrewrite.java.cleanup.ReplaceStackWithDeque
+  # We want to keep JDK16 (and not downgrade to JDK11)
+  #  - org.openrewrite.java.cleanup.ReplaceStreamToListWithCollect
```

Comment:
I think the main thing is that stream.toList is returning an immutable list

Reply:
Yeah. That might be a performance optimization. Thus, I would like too keep it. Moreover, the code is shorter.